Source/Atj/data/mvp_scenario.json

```diff
@@ -913,6 +913,30 @@
         {
             "name": "debug_player_move_to_zaks_desk_1_routine",
             "tasks": [
+                {
+                    "type": "behavior",
+                    "sync_time": "0.1",
```

Comment:
We should be able to use `0.0` now. No need to set `0.1` unless theres a specific reason.